backend/runner/taskrun/scheduler.go

```diff
@@ -1015,6 +1015,17 @@ func (s *Scheduler) getAnyProjectOwner(ctx context.Context, projectID int) (*sto
 	if err != nil {
 		return nil, err
 	}
+	// Find the project member that is not workspace owner or DBA first.
+	for _, binding := range policy.Bindings {
+		if binding.Role != api.Owner || len(binding.Members) == 0 {
+			continue
+		}
+		for _, projectMember := range binding.Members {
+			if projectMember.Role != api.Owner && projectMember.Role != api.DBA {
```

Comment:
Looks like we are comparing project role with workspace role here.

Reply:
projectMember is a UserMessage. Changed to user to make it less confusing.